pytest_reportportal/service.py

```diff
@@ -107,19 +107,14 @@ def start_pytest_item(self, test_item=None):
                       'request_body=%s', start_rq)
         self.RP.start_test_item(**start_rq)
 
-    def _get_tags(self, test_item):
+    def _get_tags(self, item):
         # try to extract names of @pytest.mark.* decorators used for test item
         # and exclude those which present in rp_ignore_tags parameter
-        mark_plugin = test_item.config.pluginmanager.getplugin('mark')
+        mark_plugin = item.config.pluginmanager.getplugin('mark')
```

Comment:
the code gating on the mark plugin shouldn't be needed

Reply:
Hi Ronny,

I remember that `mark` plug-in part of standard installation. OK, I will remove this check.
Thank you!